operators/pkg/tenant-controller/suite_test.go

```diff
@@ -47,6 +47,12 @@ var testEnv *envtest.Environment
 const targetLabelKey = "reconcile"
 const targetLabelValue = "true"
 
+const sandboxClusterRole = "crownlabs-sandbox"
+const maxConcurrentReconciles = 1
+const requeueTimeMinimum = 30 * time.Second
+const requeueTimeMaximum = 35 * time.Second
+const tenantNSKeepAlive = 80 * time.Second
```

Comment:
These constants should be set to higher values, since no requeue should happen in this test environment.

Reply:
Resolved in [c9a5fe4](https://github.com/netgroup-polito/CrownLabs/pull/821/commits/c9a5fe417574f5960036bdf84eb82b074c06a4c5)